### PR TITLE
Create requirements-dev.txt

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
issue #58

Hi @agardnerIT,
just trying to jump in this issue, please check it out

Do you wanna delete `pytest` on [requirements.txt](https://github.com/agardnerIT/tracepusher/blob/main/requirements.txt) ? I might help

### CODE REVIEW

**Comment by agardnerIT** [comment]
Thanks for the PR! Yes, please remove pytest from the standard file.

Can you also add words to the readme to explain which file to use and when?

**Comment by viktoriussuwandi** [comment]
workin on it @agardnerIT

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,2 @@
 requests==2.31.0
-pyinstaller==5.13.0
-pytest==7.4.0
-charset_normalizer<3.0
```
Comment (agardnerIT) [inline requirements.txt:4]: pytest should be the only thing removed

File: requirements-dev.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+requests==2.31.0
+pyinstaller==5.13.0
+pytest==7.4.0
```
Comment (agardnerIT) [inline requirements-dev.txt:3]: Please also add the chartset_normalizer

File: README.md [hunk 3]
```diff
@@ -104,14 +104,20 @@ docker buildx build --platform linux/arm64,linux/amd64 --push -t tracepusher:dev
 docker buildx build --platform linux/arm64,linux/amd64 --push -t tracepusher:dev -f ./docker/standard/Dockerfile .
 ```
 
-# Testing
-
-Install Pytest:
+# Install Requirement
 
+For Non-Developer
```
Comment (agardnerIT) [inline README.md]: Please pluralise: `For Non-Developers` etc.

File: README.md [hunk 4]
```diff
@@ -104,14 +104,20 @@ docker buildx build --platform linux/arm64,linux/amd64 --push -t tracepusher:dev
...
-
-Install Pytest:
+# Install Requirement
 
+For Non-Developers
 ```
 pip install -r requirements.txt
 ```
 
+For Developer
```
Comment (agardnerIT) [inline README.md:114]: For Developers

**Review by agardnerIT — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**